src/EnergyPlus/HVACInterfaceManager.hh

```diff
@@ -64,26 +64,14 @@ struct EnergyPlusData;
 
 namespace HVACInterfaceManager {
 
-    // Data
-    // MODULE PARAMETER DEFINITIONS:
     // Common Pipe Recirc Flow Directions
-    extern int const NoRecircFlow;
-    extern int const PrimaryRecirc;   // flow from Supply-outlet/Demand-inlet to Supply-inlet/demand-outlet
-    extern int const SecondaryRecirc; // flow from Supply-inlet/Demand-oulet to Supply-outlet/demand-inlet
+    constexpr int NoRecircFlow(0);
+    constexpr int PrimaryRecirc(1);   // flow from Supply-outlet/Demand-inlet to Supply-inlet/demand-outlet
+    constexpr int SecondaryRecirc(2); // flow from Supply-inlet/Demand-outlet to Supply-outlet/demand-inlet
```

Comment:
Same issue here, but this time due to the variables these are assigned to being used as reporting variables.